src/app/_services/api.service.spec.ts

```diff
@@ -88,28 +87,23 @@ describe('ApiService', () => {
 
     inject([HttpTestingController, ApiService],
       (httpMock: HttpTestingController, service: ApiService) => {
-        const mouseData = JSON.stringify({'password': testPassword,
-          'name': testUsername, 'roleId': 3
-        });
+        const postedData = JSON.stringify({ 'foo': '1Foo', 'bar': '2Bar', 'bah': '3Bah' });
+        const returnedResult = JSON.stringify( true );
 
-        service.post( testUrl, mouseData ).subscribe((data: any) => {
+        service.post( testUrl, postedData ).subscribe((data: any) => {
           expect(data).toBeDefined();
-          expect(data.name).toEqual(true);
-          expect(data.mouseArray[2]).toEqual('Squeaky');
-          expect(data.mouseArray[0]).toEqual('Gray');
-          expect(data.mouseArray[1]).toEqual('Long tail');
-          expect(data.mouseData).toEqual(mouseData);
+          expect(data).toEqual(returnedResult);
         });
 
         const req = httpMock.expectOne(baseUrl + testUrl);
         expect(req.request.method).toEqual('POST');
 
-        req.flush({'name': true, 'mouseArray': [ 'Gray', 'Long tail', 'Squeaky' ],
-          'mouseData': mouseData
-        });
+        req.flush(returnedResult);
       })
   );
 
+
+/*
```

Comment:
Just wondering why this is commented out.. the test passes, yes?

Reply:
Johnathan (@haxwell )

When I was trying to determine why the tests were failing, I was attempting to reduce complexity by commenting out the tests. Let's say that I was confused at the time and forgot to add this test back in. I just un-commented the test; it ran with no issues. After I finish fixing the login from the registration page, I'll push the changes. I'm almost done, I just need to decide how to fix one more minor issue with the interaction of registration page with login. 

Deborah